src/internal/testing.ts

```diff
@@ -5,4 +5,8 @@ export { clearOneTimeMetricsCache } from './base-component/metrics/metrics';
 export { clearMessageCache } from './logging';
 export { setGlobalFlag } from './global-flags';
 export { clearVisualRefreshState } from './visual-mode';
-export { renderWithSingleTabStopNavigation } from './single-tab-stop/__tests__/utils';
+export {
+  renderWithSingleTabStopNavigation,
+  TestSingleTabStopNavigationProvider,
+  getTestSingleTabStopNavigationProvider,
+} from './single-tab-stop/__tests__/utils';
```

Comment:
This also has to change. We exclude tests directory from the typescript production build: https://github.com/cloudscape-design/component-toolkit/blob/1bdc64ff9606237576e0b4ea78057ae77bf73ac2/tsconfig.json#L16

Currently you see it working because of some unintended behaviors in the tooling, but this is not expected.

Cross-package code should be moved outside the `__tests__` folder in something like `test-utils` if you would like to maintain a semantic name